### ISSUE TITLE
[ANDROID] Not reopen application when answer call

### ISSUE THREAD

**Issue by tunglthtv** [cybex-dev/twilio_voice#199]
Hi @cybex-dev

I can't open applicant when answer call on Android. My application is running in background. 
I am using android 11 and twilio_voice version 0.0.9.
It's seems that close the app immediately when click to answer button.
Still you get this error?

**Comment by cybex-dev** [comment]
Hi @tunglthtv 

I would highly advise you use the latest version since 0.0.9 has been deprecated and is almost 4 years old.

**Comment by cybex-dev** [comment]
Feel free to use 0.1.1 for now, make sure to
- add `MANAGE_OWN_CALLS` permission to your `AndroidManifest,xml`
- also, Android `unregister()` will not successfully unregister the user at present.

I'm preparing a release at the moment which I'll release later today fixing these issues, with enhancements.

**Comment by tunglthtv** [comment]
@cybex-dev With 0.1.1, How can i call by custom activity on Android, not use callkit system?

**Comment by cybex-dev** [comment]
@tunglthtv I have no immediate plans on re-adding this support. Maybe in the near future I'll adopt such a project.

If you need a more update-to-date `0.0.9`, might I point you to [twilio_voice_mimp](https://pub.dev/packages/twilio_voice_mimp). It is closed source, but you can update the hosted package manually and self-host on github. 

Please review changes merged into `twilio_voice` from `twilio_voice_mimp` detailed [here](https://github.com/cybex-dev/twilio_voice/blob/master/CHANGELOG.md#010).

**Comment by tunglthtv** [comment]
@cybex-dev Thanks bro

**Comment by tunglthtv** [comment]
@cybex-dev  And if i want to use callkit system of IOS. So can I use ver 0.1.1? And how can I define when have a call from Web to IOS, and show callKit system? I want to sync the outcoming call both android and ios UI/UX.

**Comment by cybex-dev** [comment]
I would strongly advise you use 0.1.3 instead, it brings a number of crucial Android fixes.

**Comment by cybex-dev** [comment]
@tunglthtv 

> And how can I define when have a call from Web to IOS, and show callKit system? I want to sync the outcoming call both android and ios UI/UX.

Could you clarify what you mean with this please.

**Comment by tunglthtv** [comment]
@cybex-dev I mean when i have an incoming call on ios, and i press answer but it doesn't pop up the ios system callkit.

**Comment by cybex-dev** [comment]
@tunglthtv this package does this automatically. 

- For **CallKit** integration, see [here](https://github.com/cybex-dev/twilio_voice/blob/master/ios/Classes/SwiftTwilioVoicePlugin.swift#L520) & [here ](https://github.com/cybex-dev/twilio_voice/blob/master/ios/Classes/SwiftTwilioVoicePlugin.swift#L549C17-L549C35) for referring code.

- For **notes and documentation**, see [here ](https://github.com/cybex-dev/twilio_voice/blob/master/README.md#features) & [here ](https://github.com/cybex-dev/twilio_voice/blob/master/README.md#ios).

**Comment by tunglthtv** [comment]
@cybex-dev I have received an incoming call. But when i press answer, i want to open as like:
![image](https://github.com/cybex-dev/twilio_voice/assets/132878153/8670cbac-843e-4145-b553-e3752b4a07c2)

**Comment by cybex-dev** [comment]
> @cybex-dev I have received an incoming call. But when i press answer, i want to open as like: ![image](https://user-images.githubusercontent.com/132878153/282234965-8670cbac-843e-4145-b553-e3752b4a07c2.png)

If I understand correctly, you want to have the same iOS callkit UI on Android.

If that is the case, you'll have to either:
1. go back to a 0.0.9 or similar version, update Android according to your liking and applies changes and fixes.
2. fork master, manually integrate a custom UI with the currently implemented `ConnectionService` API (and submit a PR if desired).

_**Please note:** CallKit & `ConnectionService` API integrate into the native system dialer, this is easier for the user to understand. CallKit and `ConnectionService` API offer extremely limited customization. I'll be looking to expand as much as the offering allows in the future._

**Comment by tunbi** [comment]
@cybex-dev i think that he want to open this screen on IOS. He received an incoming call popup. So he pressed answer button and then he want to open calling screen like the call screen system on IOS. 
He can not open this screen on IOS
![image](https://github.com/cybex-dev/twilio_voice/assets/11864847/0e8a2578-a1d0-46b7-baef-7fa12f896568)

**Comment by tunglthtv** [comment]
@cybex-dev @tunbi Yes, I want to open that screen on IOS after I accept an coming call

**Comment by tunglthtv** [comment]
Hi @cybex-dev how can i open this screen? 
![image](https://github.com/cybex-dev/twilio_voice/assets/132878153/4b050489-185a-464c-8fb7-4f77fe48c53d)

**Comment by cybex-dev** [comment]
Hi @tunglthtv 

I don't think that bit is enabled at the moment - should be possible. I'll look to add it in the next release either this week or early next week.

**Comment by cybex-dev** [comment]
It appears this might not be possible at present due to Apple's restrictive CallKit API. From observation, when the app is backgrounded then the Callkit UI will show. A workaround to the above feature is to send the app to background which will show the Callkit UI as required, otherwise the status bar Call UI will show.

https://stackoverflow.com/a/38068249

Please submit a new ticket as a feature request, I'll add this ability if/when it becomes available in the API.
